lib/rubocop/cop/rspec/rails/minitest_assertions.rb

```diff
@@ -30,66 +30,146 @@ class MinitestAssertions < Base
           RESTRICT_ON_SEND = %i[
             assert_equal
             assert_not_equal
-            refute_equal
+            assert_includes
+            assert_not_includes
             assert_nil
             assert_not_nil
+            assert_empty
+            assert_not_empty
+            refute_equal
+            refute_includes
             refute_nil
+            refute_empty
           ].freeze
 
-          # @!method minitest_equal_assertion(node)
-          def_node_matcher :minitest_equal_assertion, <<~PATTERN
+          # @!method minitest_equal(node)
+          def_node_matcher :minitest_equal, <<~PATTERN
             (send nil? {:assert_equal :assert_not_equal :refute_equal} $_ $_ $_?)
           PATTERN
 
-          # @!method minitest_nil_assertion(node)
-          def_node_matcher :minitest_nil_assertion, <<~PATTERN
+          # @!method minitest_includes(node)
+          def_node_matcher :minitest_includes, <<~PATTERN
+            (send nil? {:assert_includes :assert_not_includes :refute_includes} $_ $_ $_?)
+          PATTERN
+
+          # @!method minitest_nil(node)
+          def_node_matcher :minitest_nil, <<~PATTERN
             (send nil? {:assert_nil :assert_not_nil :refute_nil} $_ $_?)
           PATTERN
 
-          def on_send(node)
-            minitest_equal_assertion(node) do |expected, actual, fail_message|
-              prefer = replace_equal_assertion(node, expected, actual,
-                                               fail_message.first)
-              add_an_offense(node, prefer)
+          # @!method minitest_empty(node)
+          def_node_matcher :minitest_empty, <<~PATTERN
+            (send nil? {:assert_empty :assert_not_empty :refute_empty} $_ $_?)
+          PATTERN
+
+          def on_send(node) # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
```

Comment:
I've just disabled `AbcSize` for now because I don't want to go too far from #1778 until it's landed, but I think a good next step for improving this could be to move the node matchers /  block into the assertion classes too, so it becomes something like:

```
EqualAssertion.check(node)
IncludesAssertion.check(node)
NilAssertion.check(node)
EmptyAssertion.check(node)
# ...
```

(which you could probably take further by having an ASSERTION_CHECKERS array, and then the rest of this rule just iterates over that in a few places to pull through node matchers, restrict sends, `check`, etc)